nilearn/plotting/html_surface.py

```diff
@@ -127,8 +127,8 @@ def _fill_html_template(info, embed_js=True):
 
 def view_img_on_surf(stat_map_img, surf_mesh='fsaverage5',
                      threshold=None, cmap=cm.cold_hot,
-                     black_bg=False, vmax=None, colorbar=True,
-                     cbar_height=.5, cbar_fontsize=25):
+                     black_bg=False, vmax=None, vmin=None, symmetric_cmap=True,
```

Comment:
We might've had this discussion, but view_img_on_surf seems to be a public function, won't this change break it's signature?

Reply:
I don't think so because we are only appending kwargs at the end of the signature

Reply:
Yeah, dunno I confused this.